frappe/email/__init__.py

```diff
@@ -20,10 +20,10 @@ def get_contact_list(txt, page_length=20) -> list[dict]:
 	match_conditions = f"and {reportview_conditions}" if reportview_conditions else ""
 
 	out = frappe.db.sql(
-		f"""select email_id as value,
+		f"""select name as value, email_id as label,
```

Comment:
```suggestion
		f"""select email_id as value,
```

Ankush is correct

Reply:
Last time I've checked this does not work properly.